style.css

```diff
@@ -12,8 +12,10 @@ html {
 	justify-content: center;
```

Comment:
the gap changes size due to your media query when the screen shrinks (see screenshots below). Consider using `gap: 20px` or another until you're happy with the result

![image](https://user-images.githubusercontent.com/45392468/150361196-c9355d4c-277b-419d-8a38-a12bbb2c7fb3.png)

![image](https://user-images.githubusercontent.com/45392468/150361247-0fc948a4-2e99-4568-b174-df5cb5756558.png)